pyproject.toml

```diff
@@ -35,7 +35,7 @@ httpx = "^0.26.0"
 nidaqmx = "^0.9.0"
 nidmm = "^1.4.7"
 nimodinst = "^1.4.7"
-flojoy-cloud = "^0.1.3"
+flojoy-cloud = {path = "../cloud/packages/python", develop = true}
```

Comment:
Update this when `flojoy_cloud` package with the new sequencer function is publish

Reply:
Done